config/apache-ignite-zos/default-config.xml

```diff
@@ -25,17 +25,13 @@
         Alter configuration below as needed.
     -->
     <bean id="grid.cfg" class="org.apache.ignite.configuration.IgniteConfiguration">
-        <property name="localHost" value="127.0.0.1"/>
-
         <property name="discoverySpi">
             <bean class="org.apache.ignite.spi.discovery.tcp.TcpDiscoverySpi">
-                <property name="localAddress" value="127.0.0.1"/>
-
                 <property name="ipFinder">
                     <bean class="org.apache.ignite.spi.discovery.tcp.ipfinder.vm.TcpDiscoveryVmIpFinder">
                         <property name="addresses">
                             <list>
-                                <value>127.0.0.1:47500..47510</value>
+                                <value>localhost:47500..47510</value>
```

Comment:
if you want to bind to all hosts, shouldn't this be 0.0.0.0?

Reply:
This is not binding address. It should point to valid host/port of a node to discover (including itself).